src/test/java/com/google/api/codegen/testsrc/common/library.proto

```diff
@@ -754,33 +739,29 @@ message MoveBookRequest {
   // The name of the book to move.
   string name = 1 [
     (google.api.field_behavior) = REQUIRED,
-    (google.api.resource_reference) = "google.example.library.v1.Book"];
+    (google.api.resource_reference).type = "library.googleapis.com/Book"];
 
   // The name of the destination shelf.
   string other_shelf_name = 2 [
     (google.api.field_behavior) = REQUIRED,
-    (google.api.resource_reference) = "google.example.library.v1.Shelf"];
+    (google.api.resource_reference).type = "library.googleapis.com/Shelf"];
 }
 
 message ListStringsRequest {
-  string name = 1 [
```

Comment:
It would be very helpful to leave in some kind of test for `"*"`, preferably on this and the `ListStringsResponse` message types to match the library_gapic.yaml

Reply:
Ack, will do

Reply:
Punting on this in annotations.